modules/core/react/lib/InputProvider.tsx

```diff
@@ -114,6 +115,10 @@ export default class InputProvider extends React.Component<InputProviderProps, I
   private eventTimer: number | undefined;
   private ref: React.RefObject<HTMLDivElement> = React.createRef();
 
+  static defaultProps = {
+    container: document.body,
+  };
+
```

Comment:
remove

Reply:
wasn't sure yet if we wanted this to be a default

Reply:
I would expect it to be, yes 👍